pkg/cli/newapp/newapp.go

```diff
@@ -134,9 +134,16 @@ To list all local templates and image streams, use:
 To search templates, image streams, and Docker images that match the arguments provided, use:
 
   oc new-app -S php
-  oc new-app -S --template=ruby
+  oc new-app -S --template=rails
   oc new-app -S --image-stream=mysql
+  # REMINDER, for docker image references, if a registry or repository are not specified, 'docker.io' and 'library' are potential defaults for the various image registry clients
```

Comment:
I'd rather not have this warning - the root pain comes from the fact that we have a new thing that customers are using - the OpenShift Dev Sandbox. New users - who may be new to containers and image registries - are hitting this pain because our default example (that they are probably copy-pasting) hits docker.io. New users a) won't read this, and b) are probably not aware of the rate limiting, and c) are certainly not aware that the sandbox is on an OSD cluster, and as a result they are almost guaranteed to hit the rate limit.

So let's save them pain and have our example use a registry that doesn't impose such anonymous pull rate limits, like quay.io. @hhorak are there UBI-8 or Fedora-based images for python available on quay? Or are they available on a similar public registry (ex - registry.fedoraproject.org)

Reply:
> I'd rather not have this warning - the root pain comes from the fact that we have a new thing that customers are using - the OpenShift Dev Sandbox. New users - who may be new to containers and image registries - are hitting this pain because our default example (that they are probably copy-pasting) hits docker.io. New users a) won't read this, and b) are probably not aware of the rate limiting, and c) are certainly not aware that the sandbox is on an OSD cluster, and as a result they are almost guaranteed to hit the rate limit.
> 
> So let's save them pain and have our example use a registry that doesn't impose such anonymous pull rate limits, like quay.io. @hhorak are there UBI-8 or Fedora-based images for python available on quay? Or are they available on a similar public registry (ex - registry.fedoraproject.org)

there was a discussion on this back in January @adambkaplan ... I see you were not on the email thread.  I've forwarded it to you.

I'll peruse the precise inventory wrt python.

Reply:
I think the reader's digest from that thread @adambkaplan is:
- ubi8 related content will not sit in quay.io
- so we should go with the OKD version:  i.e. https://github.com/openshift/cluster-samples-operator/blob/master/assets/operator/okd-x86_64/python/imagestreams/python-centos.json#L51

Reply:
>  @hhorak are there UBI-8 or Fedora-based images for python available on quay?

All fedora images for python are in registry.fedoraproject.org only, UBI8 images are in the unauthenticated registry.access.redhat.com. We only migrated the ubi7 images to the quay.io. HTH.